### DISCUSSION TITLE
Possible way to adjust screen color.

### DISCUSSION THREAD

**Discussion by santaakos** [seerge/g-helper#1881] [Ideas]
Hi. Is there a way that you can implement a display color changing feature? It is too saturated, and without Armoury Crate I cannot change this.

**Comment by seerge** [comment]
Hello, i don't know which device you have. But you may be just missing your color profile :) so you should install it in windows.

As for Visual styles - it's addressed in FAQ
https://github.com/seerge/g-helper/wiki/FAQ#how-do-i-set-different-visual-styles

**Reply by santaakos** [reply]
Ohh. Thank you, I did not know that this software exists, my device is ROG Strix G713PI.

**Comment by seerge** [comment]
Added in latest release https://github.com/seerge/g-helper/releases
